packages/astro/src/core/render/ssr-element.ts

```diff
@@ -1,14 +1,15 @@
+import slashify from 'slash';
 import type { SSRElement } from '../../@types/astro';
-
-import npath from 'path-browserify';
-import { appendForwardSlash } from '../../core/path.js';
+import { appendForwardSlash, removeLeadingForwardSlash } from '../../core/path.js';
 
 function getRootPath(base?: string): string {
 	return appendForwardSlash(new URL(base || '/', 'http://localhost/').pathname);
 }
 
 function joinToRoot(href: string, base?: string): string {
-	return npath.posix.join(getRootPath(base), href);
+	const rootPath = getRootPath(base);
+	const normalizedHref = slashify(href);
+	return appendForwardSlash(rootPath) + removeLeadingForwardSlash(normalizedHref);
```

Comment:
This is the only place we use the `path-browserify` dep. I've emulated the posix join with this.


For reference, here's it's implementation: https://github.com/browserify/path-browserify/blob/872fec31a8bac7b9b43be0e54ef3037e0202c5fb/index.js#L180-L197